### ISSUE TITLE
Warning: The `set-output` command is deprecated

### ISSUE THREAD

**Issue by osteel** [FidelusAleksander/gh-action-regex#3]
Hi,

Thank you for the simple and useful action.

It works well but I get this warning message in the logs:

```
Warning: The `set-output` command is deprecated and will be disabled soon.
Please upgrade to using Environment Files.
For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```

Might be something to look at at some point 🙂 

Cheers,

Yannick

**Comment by FidelusAleksander** [comment]
@osteel Thanks for the feedback! 

I'm going to spend some time on improving this action sometime soon and take a look at this issue aswell

Cheers!

**Comment by FidelusAleksander** [comment]
@osteel 

This is fixed with [0.3.0 release](https://github.com/FidelusAleksander/gh-action-regex/releases/tag/v0.3.0) :)

**Comment by osteel** [comment]
@FidelusAleksander nice one! Thanks
